README.md

```diff
@@ -1 +1,141 @@
 # MediaMonks - Frontend Coding Standards
+
+[Based on Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript/)
+
+## Naming
+Choosing good names is critical to creating code that is easy to use and easy to understand. You should always take the time to think about whether you have chosen the right name for something, especially if it is part of the public API.
+
+#### English
+All code, names, comments, etc. must be in English.
+> Tip: install a spell checker in your IDE to avoid typos.
+
+#### Use one name for one thing
+Do not reuse names. Do not use names that can mean multiple things. Always use the same name for the same thing. So use the same name within the JavaScript, CSS and HTML for the same thing. Align naming with the whole team: Backend, Frontend, UX, Design, PM, client, etc.
+
+For big projects with their own jargon it could help to create a dictionary.
+
+#### Descriptive
+Names must be descriptive for the working or usage of the class, method or variable.
+
+###### The name should always end with what the object is.
+For example: a button must always end with `Button`.
+ * `MenuButton` a button used in the menu
+ * `ButtonMenu` a menu with buttons
+ * `MenuButtonIcon` an icon in the button of the menu
+ 
+There is no limit for the length of a name, so prefer a long name which is clear and descriptive than a short name which is not clear.
+
+#### Abbreviations
+Avoid them as a general rule. For example, `calculateOptimalValue()` is a better method name than `calcOptVal()`. Being clear is more important than minimizing keystrokes. If you don't abbreviate, developers won't have to remember whether you shortened a word like _qualified_ to `qual` or `qlfd`.
+
+Abbreviations that are part of the HTML, CSS and/or JavaScript API are allowed, like:
+ * `URL` for Uniform Resource Locator
+ * `URI` for Uniform Resource Identifier
+ * `src` for source
+ * `DOM` for Document Object Model
+ * `img` for image
+ 
+You should only use these abbreviations within the same context. So only use `img` if you refer to an HTML Image Tag. (`<img>`)
+
+We have standardized on a few abbreviations that are allowed to use:
+ * `app` for application
+ * `args` for arguments
+ * `auto` for automatic, as in `autoLayout`
+ * `bin` for binary
+ * `fps` for frames per second
+ * `id` for identifier. Please note that 'd' should be written in lowercase when used in combination with an other word, like `userId`.
+ * `info` for information, as in `GridRowInfo`
+ * `init` for initialize
+ * `lib` for library
+ * `max` for maximum, as in `maxHeight`
+ * `min` for minimum, as in `minWidth`
+ * `param` for parameter
+ * `params` for parameters
+ * `ref` for reference
+ * `regExp` for regular expression, as in `RegExpValidator`
```

Comment:
`RegExp` is part of the JavaScript API and can be removed from this list or moved to the other list